<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Traktana - Centro Brasileiro de Pesquisas FÃ­sicas</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>351</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>791</width>
      <height>351</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="viewMode">
     <enum>QMdiArea::SubWindowView</enum>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="subwindowStatus">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>228</width>
       <height>226</height>
      </size>
     </property>
     <property name="windowTitle">
      <string>Status</string>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <widget class="QLabel" name="labelTime">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCycles">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cycles:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPlain">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Length plain:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditTime">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCycles">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPlain">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>211</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="subwindowOperation">
     <property name="minimumSize">
      <size>
       <width>249</width>
       <height>202</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="windowTitle">
      <string>Operation</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>42</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>75</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>108</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>77</x>
        <y>9</y>
        <width>43</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>77</x>
        <y>42</y>
        <width>63</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
      <property name="geometry">
       <rect>
        <x>77</x>
        <y>75</y>
        <width>63</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
      <property name="geometry">
       <rect>
        <x>77</x>
        <y>108</y>
        <width>63</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="subwindowDebug">
     <property name="minimumSize">
      <size>
       <width>266</width>
       <height>195</height>
      </size>
     </property>
     <property name="windowTitle">
      <string>Debug</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuToolBars">
     <property name="title">
      <string>ToolBars</string>
     </property>
     <addaction name="actionOperationTool"/>
     <addaction name="actionDeviceTool"/>
    </widget>
    <addaction name="actionOperation"/>
    <addaction name="actionStatus"/>
    <addaction name="actionDebug"/>
    <addaction name="separator"/>
    <addaction name="menuToolBars"/>
   </widget>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>Operation</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionPause"/>
    <addaction name="actionRestart"/>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionConnect_Debug"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionSerialDeviceConfig"/>
    <addaction name="actionSerialDebugConfig"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuOperation"/>
   <addaction name="menuDevice"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarOperation">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionRestart"/>
  </widget>
  <widget class="QToolBar" name="toolBarConnect">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionSerialDeviceConfig"/>
   <addaction name="actionConnect_Debug"/>
   <addaction name="actionSerialDebugConfig"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOperation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Operation</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionOperationTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Operation</string>
   </property>
  </action>
  <action name="actionDeviceTool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Device/Configuration</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset>
     <normalon>icons/play.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/pause.png</normaloff>icons/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="icon">
    <iconset>
     <normaloff>icons/refresh_32.png</normaloff>icons/refresh_32.png</iconset>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset>
     <normaloff>icons/connect.png</normaloff>icons/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionConnect_Debug">
   <property name="icon">
    <iconset>
     <normaloff>icons/connectbug.png</normaloff>icons/connectbug.png</iconset>
   </property>
   <property name="text">
    <string>Connect Debug</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionSerialDeviceConfig">
   <property name="icon">
    <iconset>
     <normaloff>icons/cog.png</normaloff>icons/cog.png</iconset>
   </property>
   <property name="text">
    <string>Serial Device</string>
   </property>
  </action>
  <action name="actionSerialDebugConfig">
   <property name="icon">
    <iconset>
     <normaloff>icons/cogbug.png</normaloff>icons/cogbug.png</iconset>
   </property>
   <property name="text">
    <string>Serial Debug</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset>
     <normaloff>icons/disconnect.png</normaloff>icons/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionDisconnectDebug">
   <property name="icon">
    <iconset>
     <normaloff>icons/disconnectbug.png</normaloff>icons/disconnectbug.png</iconset>
   </property>
   <property name="text">
    <string>DisconnectDebug</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
